libft/tests/map/mapss/ft_mapss_add_test.c

```diff
@@ -39,6 +39,7 @@ static void	generate_tests(t_testcase *tests)
 	tests[6] = (t_testcase){"OLDPWD", ":0", 6, {"HOME", "OLDPWD", "PATH", "SHELL", "SHLVL", "USERNAME"}, {"/home/user", ":0", "/usr/local/sbin", "/bin/bash", "1", "/usr/local/sbin"}};
 	tests[7] = (t_testcase){"DISPLAY", "X Window", 7, {"DISPLAY", "HOME", "OLDPWD", "PATH", "SHELL", "SHLVL", "USERNAME"}, {"X Window", "/home/user", ":0", "/usr/local/sbin", "/bin/bash", "1", "/usr/local/sbin"}};
 	tests[8] = (t_testcase){"", "Empty_sring", 8, {"", "DISPLAY", "HOME", "OLDPWD", "PATH", "SHELL", "SHLVL", "USERNAME"}, {"Empty_sring", "X Window", "/home/user", ":0", "/usr/local/sbin", "/bin/bash", "1", "/usr/local/sbin"}};
+	tests[9] = (t_testcase){"DISPLAY", "nuffin", 8, {"", "DISPLAY", "HOME", "OLDPWD", "PATH", "SHELL", "SHLVL", "USERNAME"}, {"Empty_sring", "nuffin", "/home/user", ":0", "/usr/local/sbin", "/bin/bash", "1", "/usr/local/sbin"}};
```

Comment:
why did you add this test?

Reply:
To check that a value is overwritten if it correspoNDS to an existing key